File: remove_tenant_traits.php
<?php

$dir = __DIR__ . '/app/Models';
$files = new RecursiveIteratorIterator(new RecursiveDirectoryIterator($dir));

foreach ($files as $file) {
    if ($file->isFile() && $file->getExtension() === 'php') {
        $path = $file->getRealPath();
        $content = file_get_contents($path);
        
        if (strpos($content, 'BelongsToTenant') !== false) {
            $lines = explode("\n", $content);
            $newLines = [];
            foreach ($lines as $line) {
                if (strpos($line, 'use App\\Traits\\BelongsToTenant;') !== false) continue;
                if (trim($line) === 'use BelongsToTenant;') continue;
                $newLines[] = $line;
            }
            file_put_contents($path, implode("\n", $newLines));
            echo "Removed BelongsToTenant from " . basename($path) . "\n";
        }
    }
}

File: remove_tenant_traits.py
import os

model_dir = r"d:\Code\erp-single-tenant\app\Models"

for root, _, files in os.walk(model_dir):
    for file in files:
        if file.endswith(".php"):
            filepath = os.path.join(root, file)
            try:
                with open(filepath, "r", encoding="utf-8") as f:
                    content = f.read()
            except Exception as e:
                print(f"Error reading {file}: {e}")
                continue
            
            if "BelongsToTenant" in content:
                lines = content.split('\n')
                new_lines = []
                for line in lines:
                    if 'use App\\Traits\\BelongsToTenant;' in line:
                        continue
                    # Match use BelongsToTenant; inside class
                    if line.strip() == 'use BelongsToTenant;':
                        continue
                    new_lines.append(line)
                
                with open(filepath, "w", encoding="utf-8") as f:
                    f.write('\n'.join(new_lines))
                print(f"Removed BelongsToTenant from {file}")
